src/com/base/game/character/body/OrificeUrethra.java

```diff
@@ -149,27 +149,27 @@ public String setCapacity(GameCharacter owner, float capacity) {
 			if (owner.isPlayer()) {
 				return "<p>"
 							+ "You let out a shocked gasp as you feel your urethra dilate and stretch out as its internal [style.boldGrow(capacity increases)].</br>"
-							+ "The transformation quickly pcockes, leaving you with [style.boldSex(" + UtilText.generateSingluarDeterminer(capacityDescriptor) + " " + capacityDescriptor + " urethra)]!"
```

Comment:
Oh dear, that must have happened when I replaced all instances of "ass" with "cock"...